404.html

```diff
@@ -1,67 +1,121 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Page not found - Hubble Homes</title>
+    <script type="text/javascript">
+      window.isErrorPage = true;
+      window.errorCode = '404';
+    </script>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta property="og:title" content="Page not found" />
+    <script
+      src="/scripts/scripts.js"
+      type="module"
+      crossorigin="use-credentials"
+    ></script>
+    <script src="/scripts/aem.js" type="module"></script>
+    <link rel="stylesheet" href="/styles/styles.css" />
+    <style>
+      main.error {
+        min-height: calc(100vh - var(--nav-height));
+        display: flex;
+        align-items: center;
```

Comment:
Remove the center alignment for the items and it should bring the items to the top of the page as per the discussion with Nate.

```suggestion
```

Reply:
changed